wrangler-core/src/main/java/io/cdap/wrangler/expression/EL.java

```diff
@@ -129,31 +145,25 @@ public ELResult execute(ELContext context) throws ELException {
     public Map<String, Object> functions() {
       Map<String, Object> functions = new HashMap<>();
       functions.put(null, Global.class);
-      functions.put("date", Dates.class);
-      functions.put("json", JSON.class);
-      functions.put("math", Math.class);
-      functions.put("string", StringUtils.class);
```

Comment:
do we not need this function? Is there any impact on directives being applied due to removal of these functions?

Reply:
DIdn't remove, just changed the classes and separated them correctly.